eng/common/instructions/azsdk-tools/check-package-validation.instructions.md

```diff
@@ -0,0 +1,22 @@
+---
+description: 'Check if the package validation checks have passed for the SDK package.'
+---
+## Goal
+Check the validation checks for the SDK package by collecting the required information from the user and executing the `azsdk_package_run_check` tool.
+
+## Instructions
+1. **Collect Required Information**:
+    - Prompt the user for the absolute package path to the package directory for the SDK package they want to check.
```

Comment:
Isn't this tool triggered for the package in current repo in context? Why does this need to be absolute path? We should check this can be simplified to take just package name or relative path to package in one of the future enhancements.